src/garage/torch/algos/vpg.py

```diff
@@ -11,6 +11,22 @@
 from garage.torch.algos import loss_function_utils as loss_utils
 
 
+def _add_padding_2d(vals, length):
+    tensor = torch.Tensor(vals)
+    padding_length = max(length - tensor.shape[0], 0)
+    return F.pad(tensor, (0, 0, 0, padding_length))
+
+
+def _add_padding_1d(vals, length):
```

Comment:
i think these would be more clearly named as something like `pad_to_length` this makes it more clear how this is different than the default `F.pad`.

Reply:
Changed the name into `pad_to_last` since the method only put the padding numbers on the last